optaplanner-website-docs/supplemental-ui/partials/header-content.hbs

```diff
@@ -2,14 +2,20 @@
   <nav class="navbar">
     <div class="navbar-brand">
       <div class="navbar-item">
-        <a href="{{{uiRootPath}}}/../../">
+        <a href="{{{or site.url siteRootPath}}}">
```

Comment:
This changes the link from `www.optaplanner.org` to `www.optaplanner.org/docs`. The logo should point back to the OptaPlanner website.

Reply:
Fixed